mockup-contributions_list.php

```diff
@@ -0,0 +1,922 @@
+<?php
```

Comment:
Okay sounds good. I think we're going to need some further information on the readme + I'll create a couple of tasks in csio for adding a 'components' and 'common classes' demo page.